concordium-std-derive/CHANGELOG.md

```diff
@@ -6,3 +6,4 @@
 
 - Add macros for deriving error implementations.
 - Make derive macros slightly more hygienic.
+- Enable renaming of enum variants and field names.
```

Comment:
You should probably also mention that this is in regard to schemas.

Reply:
Done.